modules/certmanager/certificate.go

```diff
@@ -66,10 +67,15 @@ func NewCertificate(
 	certificate *certmgrv1.Certificate,
 	timeout time.Duration,
 ) *Certificate {
-	return &Certificate{
+	crt := &Certificate{
 		certificate: certificate,
 		timeout:     timeout,
 	}
+
+	sort.Strings(crt.certificate.Spec.IPAddresses)
```

Comment:
Sorting as strings will not order by IP address correctly. Could we parse the IPs and then sort (as byte slices I guess)?

Reply:
right, let me fix that

Reply:
updated and added an ip sort func